### ISSUE TITLE
Specify 'use_bias=True' explicitly, the model get convergence; but use default settings couldn't

### ISSUE THREAD

**Issue by 1165048017** [keras-team/keras#14213]
I tried to build a model with one Hidden Layer (2 units) for XOR operation
when I use this code to build the model: 

```
inputs = keras.Input(shape=[2,])
x= keras.layers.Dense(2,activation='tanh')(inputs)
x = keras.layers.Dense(1,activation='sigmoid')(x)
model1 = keras.models.Model(inputs=inputs,outputs=x)
```

It couldn't get convergence,the loss and accuracy curves:
![26-1](https://user-images.githubusercontent.com/15133787/92895763-5fcb3200-f44e-11ea-9ad6-c8ae7d61b4a0.png)

but when I use:
```
inputs = keras.Input(shape=[2,])
x= keras.layers.Dense(2,activation='tanh',use_bias=True)(inputs)
x = keras.layers.Dense(1,activation='sigmoid')(x)
model2 = keras.models.Model(inputs=inputs,outputs=x)
```
The mode could get convergence very quickly
![26-2](https://user-images.githubusercontent.com/15133787/92895774-635eb900-f44e-11ea-968c-42da6b16abd9.png)

the original code is here:https://drive.google.com/file/d/1yLhzvgqRprbPOvOT5BRD01C7Z_YMkptz/view?usp=sharing

The environment is in colab

**Comment by 1165048017** [comment]
The issue maybe caused by the local optimum~~~~

**Comment by sushreebarsa** [comment]
@1165048017 Is this still an issue?
In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by sushreebarsa** [comment]
@1165048017 Please refer to this [link](https://keras.io/api/layers/core_layers/dense/) and let us know if it helps?Thanks!
